src/components/App/App.jsx

```diff
@@ -93,44 +130,41 @@ function App() {
   };
 
   const handleSignIn = ({ email, password }) => {
-    authorize({ email, password })
-      .then(() => {
-        checkToken()
-          .then((res) => {
-            setCurrentUser({
-              name: res.data.name,
-              email: res.data.email,
-              _id: res.data._id,
-            });
-            setIsLoggedIn(true);
-            onClose();
-          })
-          .catch((err) => {
-            console.log(err);
-          });
-      })
-      .catch((err) => {
-        console.log(err);
+    if (!email || !password) {
+      return;
+    }
+    return authorize(email, password).then((data) => {
+      tokenValue.setToken(data.token);
+      return getUserInfo(data.token).then((res) => {
+        setIsLoggedIn(true);
+
+        setCurrentUser({
+          name: res.data.username,
+          email: res.data.email,
+          _id: res.data._id,
+        });
       });
+    });
   };
 
```

Comment:
* Please, remove one extra `App.jsx` from the project https://snipboard.io/1K4e5W.jpg . It's misleading because they have different code